modules/rgbd/src/normal.cpp

```diff
@@ -836,6 +837,7 @@ multiply_by_K_inv(const Matx<T, 3, 3> & K_inv, U a, U b, U c, Vec<T, 3> &res)
 
     // Get the normals
     normals_out.create(points3d_ori.size(), CV_MAKETYPE(depth_, 3));
+    normals_out.setTo(0);
```

Comment:
ok, it matters but only for linemod so you should actually set that setTo only before https://github.com/Itseez/opencv_contrib/blob/master/modules/rgbd/src/normal.cpp#L437  (it's just a small performance hit to do a getTo when not needed).